### PR TITLE
add new mtx commands

### PR DESCRIPTION
**Pull request by DarkPark**

### CODE REVIEW

File: src/mtx/commands/uplink/GetDeviceTypeResponse.ts [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+const COMMAND_SIZE = 9;
+
+const examples: TCommandExampleList = [
+    {
+        name: 'type 1',
+        parameters: {
+            type: 'MTX 1A10.DG.2L5-LD4',
+            flags: 255
+        },
+        hex: {header: '04 09', body: '00 11 21 49 21 B6 81 C0  ff'}
```
Comment (DeadByElpy) [inline src/mtx/commands/uplink/GetDeviceTypeResponse.ts:23]:
```suggestion
        hex: {header: '04 09', body: '00 11 21 49 21 B6 81 C0 ff'}
```

File: src/mtx/commands/uplink/GetDeviceTypeResponse.ts [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    },
+    {
+        name: 'type 1',
+        parameters: {
+            type: 'MTX 1G05.DH.2L2-DOB4',
+            flags: 0
+        },
+        hex: {header: '04 09', body: '00 12 16 47 21 B3 17 2C  00'}
```
Comment (DeadByElpy) [inline src/mtx/commands/uplink/GetDeviceTypeResponse.ts:31]:
```suggestion
        hex: {header: '04 09', body: '00 12 16 47 21 B3 17 2C 00'}
```

File: src/mtx/commands/uplink/index.ts [hunk 3]
```diff
@@ -7,12 +7,19 @@
 export {default as ActivateRatePlanResponse} from './ActivateRatePlanResponse.js';
 export {default as ErrorResponse} from './ErrorResponse.js';
 export {default as GetDateTimeResponse} from './GetDateTimeResponse.js';
+export {default as GetDayProfileResponse} from './GetDayProfileResponse.js';
 export {default as GetDeviceIdResponse} from './GetDeviceIdResponse.js';
+//export {default as GetDeviceTypeResponse} from './GetDeviceTypeResponse.js';
```
Comment (DeadByElpy) [inline src/mtx/commands/uplink/index.ts:12]: command not finished yet?

Comment (DarkPark) [reply src/mtx/commands/uplink/index.ts:12]: yes

File: src/mtx/commands/uplink/GetDeviceTypeResponse.ts (hunk 1 above)
Comment (DarkPark) [reply src/mtx/commands/uplink/GetDeviceTypeResponse.ts:23]: done

File: src/mtx/commands/uplink/GetDeviceTypeResponse.ts (hunk 2 above)
Comment (DarkPark) [reply src/mtx/commands/uplink/GetDeviceTypeResponse.ts:31]: done

**Review by DeadByElpy — APPROVED** [review]

**Pull request merged**